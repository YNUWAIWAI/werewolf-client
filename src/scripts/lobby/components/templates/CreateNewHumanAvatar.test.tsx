import * as React from 'react'
import {CreateNewHumanAvatar} from './CreateNewHumanAvatar'
import {shallow} from 'enzyme'

test('render', () => {
  const wrapper = shallow(
    <CreateNewHumanAvatar
      command={[]}
      menuItems={[]}
    />
  )

  expect(wrapper.getElement()).toMatchSnapshot()
})
