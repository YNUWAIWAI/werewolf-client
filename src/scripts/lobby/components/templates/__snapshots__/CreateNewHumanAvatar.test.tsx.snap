// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  className="lo--grid"
>
  <Header
    id="Header.selectHumanAvatar"
  />
  <MainContent>
    <CreateNewAvatarHeader />
    <Memo(Connect(CreateNewHumanAvatarTable)) />
    <Memo(Connect(Menu))
      className="lo--create-new-avatar--command"
      itemClassName="lo--create-new-avatar--command--item"
      items={Array []}
    />
  </MainContent>
  <AsideContent>
    <Memo(Connect(Menu))
      className="lo--compact-menu"
      itemClassName="lo--compact-menu--item"
      items={Array []}
    />
  </AsideContent>
</div>
`;
